src/common/Widget.js

```diff
@@ -43,6 +43,25 @@
         this._watchArr = [];
 
         this._renderCount = 0;
+
+        this._hpcc_debug = false;
+        var context = this;
+        var def = window.location.search.split("?")[1];
+        if (def) {
+            def.split("&").forEach(function (param, idx) {
+                var paramParts = param.split("=");
+                if (decodeURIComponent(paramParts[0]) === 'hpcc_debug' && (decodeURIComponent(paramParts[1]) === "true" || decodeURIComponent(paramParts[1]) === "1")) {
+                    context._hpcc_debug = true;
+                }
+            });
+        }
+
+        if (requirejs.s.contexts._.config.__hpcc_debug || this._hpcc_debug || window._hpcc_debug) { // jshint ignore:line
```

Comment:
Just make this ```if (window.__hpcc_debug)``` then the developer has many different ways to sat it if needs be.